open_academy/models/course.py

```diff
@@ -8,3 +8,5 @@ class Course(models.Model):
     name = fields.Char(required=True)
     title = fields.Text()
     description = fields.Text()
+    responsible_user_id = fields.Many2one('res.users')
+    session_id = fields.One2many('session', 'courses_id')
```

Comment:
This field is one2many, you need to set the `_ids` so we know that that field will have multiple records

```suggestion
    session_ids = fields.One2many('session', 'courses_id')
```